### PR TITLE
in progress, style clinics index to accordion collapse

### PR DESCRIPTION
**Pull request by SamuelLangenfeld**
I rule and have completed some work on Case Manager that's ready for review!

(brief, plain english overview of your changes here)

This pull request makes the following changes:
* (your change here)
* (another change here)
* (etc)

(If there are changes to the views, please include a screenshot so we know what to look for!)

It relates to the following issue #s: 
* Fixes #X
* Bumps #947

### CODE REVIEW

**Comment by colinxfleming** [comment]
:eagle: 

thanks a ton for this, @SamuelLangenfeld ! I'll keep an eye on this, HMU if you need a hand pushing it over.

**Comment by colinxfleming** [comment]
How's this feeling @SamuelLangenfeld ?

**Comment by SamuelLangenfeld** [comment]
Hey @colinxfleming, it looks good now. You can check it out but I think it's ready.

**Comment by colinxfleming** [comment]
Also, wanted to check -- in the PR it says this bumps the issue. Is there anything left to do on the issue after this PR gets merged, or can we consider this fixed?

File: app/views/clinics/index.html.erb [hunk 1]
```diff
@@ -1,53 +1,56 @@
...
-              </tbody>
-            </table>
-          </div>
-        </div>
-      </div>
-      <a role="button" class="clinic-detail-toggle" data-toggle="collapse" href="#<%= clinic.id %>" aria-expanded="false" aria-controls="<%= clinic.id %>">More details</a>
-    </div>
-  </div>
-<% end %>
+        </td>
```
Comment (colinxfleming) [inline app/views/clinics/index.html.erb]: Turns out this is complicated as all hell! Let's punt on it.

**Pull request merged**